runelite-client/src/main/java/net/runelite/client/plugins/experiencedrop/XpDropConfig.java

```diff
@@ -83,10 +83,33 @@ default Color getMagePrayerColor()
 		description = "Configures how many ticks should pass between fake XP drops, 0 to disable",
 		position = 4
 	)
-	@Units(Units.TICKS)
 	default int fakeXpDropDelay()
 	{
 		return 0;
 	}
 
+	@Units(Units.TICKS)
+
+	@ConfigItem(
+		keyName = "overrideClientColor",
+		name = "Override XP Drop Color",
+		description = "Configure if you wish to override the XP drop color set in game",
+		position = 5
+	)
+	default boolean overrideColor()
+	{
+		return false;
+	}
+
+	@ConfigItem(
+		keyName = "standardColor",
+		name = "Standard Color",
+		description = "XP drop color when no prayer is active",
+		position = 6
+	)
+	default Color standardColor()
```

Comment:
You can just do `Color standardColor()` and if it is null not apply it, which removes the need for a separate on/off toggle.

Reply:
I tried the following code snippets...
### Config:
```java 	
        @ConfigItem(
		keyName = "standardColor",
		name = "Standard Color",
		description = "XP drop color when no prayer is active",
		position = 6
	)
	default Color standardColor()
	{
		return null;
	}
```
### Plugin:
```java Color standardColor = config.standardColor();

		if (standardColor != null)
		{
			widget.setTextColor(standardColor.getRGB());
		}
		else
		{
			EnumComposition colorEnum = client.getEnum(EnumID.XPDROP_COLORS);
			int defaultColorId = config.standardColor().getRGB();
			int color = colorEnum.getIntValue(defaultColorId);
			widget.setTextColor(color);
		}
```

But I couldn't figure out how the user would completely disable or be able to reset the Standard Color. I have set it to null by default, but even when resetting the plugin to it's default config, the Standard Color does not reset.

I have also removed the Override Color bool toggle as you suggested.

Any idea how I might be able to achieve this?

Reply:
It looks like you're not doing the `else` part correctly. In fact, in your snippet, it would cause a NullPointerException on `int defaultColorId = config.standardColor().getRGB();`. The `else` part should be the unchanged old logic.

Try something like this.
```java
private void resetTextColor(Widget widget)
{
	Color standardColor = config.standardColor();
	if (standardColor != null)
	{
		widget.setTextColor(standardColor.getRGB());
	}
	else
	{
		EnumComposition colorEnum = client.getEnum(EnumID.XPDROP_COLORS);
		int defaultColorId = client.getVar(Varbits.EXPERIENCE_DROP_COLOR);
		int color = colorEnum.getIntValue(defaultColorId);
		widget.setTextColor(color);
	}
}
```

Reply:
I've pushed a fixed commit with this fixed.